include/nfc/tnep/ch.rst

```diff
@@ -0,0 +1,92 @@
+.. _nfc_tnep_ch_readme:
+
+NFC TNEP Connection Handover
+############################
+
+The NFC TNEP Connection Handover service handles the exchange of Connection Handover Messages between an NFC Forum Tag and an NFC Forum Poller device.
+It uses the TNEP protocol in the Single Response Communication mode to exchange the handover messages between:
+
+   * NFC Tag Device (see :ref:`tnep_tag_readme`)
+   * NFC Poller Device (see :ref:`tnep_poller_readme`)
+
+The Connection Handover Service name URI for the service announced in the Service Parameter record is ``urn:nfc:sn:handover``.
+
+Handover protocol
+*****************
+
+The Connection Handover Protocol enables two NFC Forum devices to negotiate a shared set of alternative (non-NFC) communication carriers.
+The Handover Requestor Device is the NFC Forum Device that initiates the handover negotiation.
```

Comment:
Can you please check in the spec whether this should be Requestor or Requester. I'm not sure, I can see it spelled both ways everywhere and it was also spelled in both ways here. For now, I changed all occurrences to 'Requestor'.

Reply:
Yes sure. It seems like everywhere in spec this is called the Requester

Reply:
I chnaged it everywhere to the Requester